### DISCUSSION TITLE
Enabling H-extension

### DISCUSSION THREAD

**Discussion by FrancoAuri96** [eugene-tarassov/vivado-risc-v#190] [Q&A]
Hello Eugene,
my goal is to run Xvisor hypervisor on this project, but i need the H-extension to make it work.
Is it possible to configure the rocket chip generation, directly from this project, in order to implement the H-extension?

**Comment by eugene-tarassov** [comment]
Yes, it is possible to configure the rocket chip generation by editing [src/main/scala/rocket.scala](https://github.com/eugene-tarassov/vivado-risc-v/blob/master/src/main/scala/rocket.scala).
Also, see #184

**Comment by FrancoAuri96** [comment]
ok, thank you Eugene!
